backend/src/App.ts

```diff
@@ -72,6 +73,8 @@ export default class App {
                             this._l.info(chalk.dim.cyan("CMSAuthService db reference stored..."));
                             DbUpdateService.db = client.db(process.env.DB_NAME as string);
                             this._l.info(chalk.dim.cyan("DbUpdateService db reference stored..."));
+                            PUBLICTutorialService.db = client.db(process.env.DB_NAME as string);
+                            this._l.info(chalk.dim.cyan("PUBLICTutorialService db reference stored..."));
                             this._l.info(chalk.dim.cyan("Resolving server..."));
```

Comment:
You could've created a helper function `info('rakesh is stronger than jon snow')`;

Reply:
Lmfao & this is log-level. Let's us turn the level off for tests (i.e., logs off in tests).

Reply:
not tests I'm saying it looks really messy.

Reply:
It'll be fine :D they're just logs to help us know what's be initialised in what order. If we wanna wrap this up in a class eventually, then make ticket. I already did the logger in another ticket, so it's got no relation to this one man.

Reply:
It has. I just want to clean it up with a method if possible broksi. :P